### PR TITLE
J-259 + fixed shifting markup when scroll appears

### PR DESCRIPTION
**Pull request by vitalik-prog**

### CODE REVIEW

**Comment by vitalik-prog** [comment]
[Fix header markup. When clicked on bell icon markup shifts to the right](https://trello.com/c/NqWs1sU5/259-fix-header-markup-when-clicked-on-bell-icon-markup-shifts-to-the-right)

**Review by Denidzio — DISMISSED** [review]

File: frontend/src/assets/css/scaffolding.scss [hunk 1]
```diff
@@ -5,7 +5,20 @@
 }
 
 html {
-  scroll-behavior: smooth;
+  width:100vw;
```
Comment (what1s1ove) [inline frontend/src/assets/css/scaffolding.scss]:
```suggestion
```

File: frontend/src/assets/css/scaffolding.scss [hunk 2]
```diff
@@ -5,7 +5,20 @@
 }
 
 html {
-  scroll-behavior: smooth;
```
Comment (what1s1ove) [inline frontend/src/assets/css/scaffolding.scss:8]: return it

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Comment by what1s1ove** [comment]
now it is not needed

**Pull request closed without merging**